### PR TITLE
(low priority) Add port fwding info to Tilt doc

### PR DESCRIPTION
**Pull request by carlisia**
Signed-off-by: Carlisia <carlisia@vmware.com>

### CODE REVIEW

File: site/content/docs/main/tilt.md [hunk 1]
```diff
@@ -129,13 +129,17 @@ spec:
...
 Here are two ways to use MinIO as the storage:
 
-1) As a MinIO instance running inside your cluster
+1) As a MinIO instance running inside your cluster (don't do this for production!)
 
     In the `tilt-settings.json` file, set `"setup-minio": true`. This will configure a Kubernetes deployment containing a running
-instance of Minio inside your cluster. There are [extra steps](contributions/minio/#expose-minio-outside-your-cluster-with-a-service)
-necessary to expose Minio outside the cluster. Note: with this setup, when your cluster is terminated so is the storage and any backup/restore in it.
+instance of MinIO inside your cluster. There are [extra steps](contributions/minio/#expose-minio-outside-your-cluster-with-a-service)
+necessary to expose Minio outside the cluster.
```
Comment (jenting) [inline site/content/docs/main/tilt.md:136]:
```suggestion
necessary to expose MinIO outside the cluster. 
```

File: site/content/docs/v1.5/tilt.md [hunk 2]
```diff
@@ -128,13 +129,17 @@ spec:
...
 Here are two ways to use MinIO as the storage:
 
-1) As a MinIO instance running inside your cluster
+1) As a MinIO instance running inside your cluster (don't do this for production!)
 
     In the `tilt-settings.json` file, set `"setup-minio": true`. This will configure a Kubernetes deployment containing a running
-instance of Minio inside your cluster. There are [extra steps](contributions/minio/#expose-minio-outside-your-cluster-with-a-service)
-necessary to expose Minio outside the cluster. Note: with this setup, when your cluster is terminated so is the storage and any backup/restore in it.
+instance of MinIO inside your cluster. There are [extra steps](contributions/minio/#expose-minio-outside-your-cluster-with-a-service)
+necessary to expose Minio outside the cluster.
```
Comment (jenting) [inline site/content/docs/v1.5/tilt.md:136]:
```suggestion
necessary to expose MinIO outside the cluster. 
```

**Review by nrb — COMMENTED** [review]
I'm ok with this, but let's incorporate JenTing's feedback for consistency.

**Comment by carlisia** [comment]
Thanks for this correction!

Ready for review.

**Review by jenting — APPROVED** [review]

**Review by zubron — APPROVED** [review]
Thank you!

**Pull request merged**